src/main/java/Duke.java

```diff
@@ -1,10 +1,205 @@
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    enum Command {
+        LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE, BYE
     }
+    public static void main(String[] args) throws IOException {
+        ArrayList<Task> list = new ArrayList<>();
+
+        // Load tasks from file
+        // Solution below inspired by https://stackoverflow.com/questions/16111496/java-how-can-i-write-my-arraylist-to-a-file-and-read-load-that-file-to-the
+        try (ObjectInputStream ois = new ObjectInputStream(new FileInputStream("./data/duke.txt"))) {
+            list = (ArrayList<Task>) ois.readObject();
+        } catch (IOException | ClassNotFoundException e) {
+            try {
+                // make directory first
+                new File("./data").mkdirs();
+                // then make file
+                // Solution below inspired by https://www.w3schools.com/java/java_files_create.asp
+                File myObj = new File("./data/duke.txt");
+                if (myObj.createNewFile()) {
+                    System.out.println("File created: " + myObj.getName());
+                } else {
+                    System.out.println("File already exists.");
+                }
+            } catch (IOException i) {
+                System.out.println("An error occurred.");
+                e.printStackTrace();
+            }
+            System.out.println("Could not load tasks from file: " + e.getMessage());
+        }
+
+        // Start-up introduction
+        printBreak();
+        System.out.println("Hello! I'm Klara");
+        System.out.println("What can I do for you?");
+        printBreak();
+
+        Scanner sc = new Scanner(System.in);
+        String word = sc.nextLine();
```

Comment:
I think `word` is not the best variable name here, since we are reading in a line which can contain multiple words, consider `userInput` as a possible alternative